code.test.js

```diff
@@ -75,12 +77,7 @@ const testCases = [
 for(let index = 0; index < testCases.length; index++){
     const { graph1, graph2, expected } = testCases[index];
     const result = are_isomorphic(graph1, graph2);
-
-    if (result !== expected) {
-        console.log(`Test Case ${index + 1}: Failed`);
-        console.log("Tests failed");
-        break;
-    } else {
-        console.log(`Test case ${index + 1}: Passed`);
-    }
+    //here is the modifed test I was missing
+    console.log(`Expected: ${expected}, Actual: ${result}`);
+    assert(result, expected);
```

Comment:
You need a Boolean statement here, i.e. `assert(result === expected)`.

Reply:
I'm so sorry I just implemented the assert(result ===expected). I think thats updated correctly now